Yoast/wpseo-news
# Replace HelpScout beacon

## Summary

This PR can be summarized in the following changelog entry:

* Replaces the HelpScout beacon on the News SEO admin page.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Go to the News admin page and see that HelpScout is working.

Fixes #561
